index.mjs

```diff
@@ -0,0 +1,62 @@
+//. # Fluture Node
+//.
+//. Common Node API's wrapped to return [Fluture][] Futures.
+//.
+//. ## API
+
+import Future from 'fluture';
+
+//# once :: String -> EventEmitter -> Future Error a
+//.
+//. Resolve a Future with the first event emitted over
+//. the given event emitter under the given event name.
+//.
+//. When the Future is cancelled, it removes any trace of
+//. itself from the event emitter.
```

Comment:
There are currently no code examples, as each of these functions return Futures. Before creating these examples, I would want to discuss how doctest could assert equality of Futures.

Reply:
Since doctests can't run on mjs files anyway, I will consider this a problem for my future self.